go.mod

```diff
@@ -3,44 +3,14 @@ module sigs.k8s.io/cluster-api-provider-docker
 go 1.12
 
 require (
-	github.com/appscode/jsonpatch v0.0.0-20190108182946-7c0e3b262f30 // indirect
```

Comment:
curious why all these are disappearing

Reply:
It's only because I deleted the code that used them. Some form of this will come back when we need it.